### DISCUSSION TITLE
ETA for next release version after 0.4.0?

### DISCUSSION THREAD

**Discussion by vinhdizzo** [duckdb/duckdb#4554] [General]
Hi,

Couldn't find this information, does anyone know of the time frame?  Asking as I'm writing an R package that will leverage duckdb, and things work with the current master version but not 0.4.0.  I'd like to time it with the next release of duckdb.  Thank you.

**Comment by Mause** [comment]
I believe it's going to be September 5th?

**Comment by vinhdizzo** [comment]
Thanks @Mause

**Reply by Mause** [reply]
It's now been released
